frontend/src/components/FullWidthAlert.tsx

```diff
@@ -22,15 +23,16 @@ const getBGColor = (type: Props["type"]) => {
   }
 };
 
-const FullWidthAlert = ({ type, children }: Props) => {
+const FullWidthAlert = ({ type, heading, children }: Props) => {
   return (
     <div className={`${getBGColor(type)}`}>
-      <GridContainer className="desktop:padding-y-2 tablet-lg:padding-y-1">
+      <GridContainer className="padding-y-1 tablet-lg:padding-y-2">
         <Grid>
           <USWDSAlert
             className="border-left-0 bg-transparent padding-left-0 margin-x-neg-2"
             type={type}
-            headingLevel="h4"
+            headingLevel="h5"
```

Comment:
You can pass the Truss component a heading level, but don't think you can it specific class to override the style. This is unfortunate, and less semantic, but an `h4` is too large visually.

Reply:
> "Heading levels should only increase by one (heading-order)"

Ugh. I'll have to fix this.

Reply:
Yup. You could also consider keeping it too big with the h4 for now and dealing with it in another pr if you want to take more time with it.

Reply:
Also, I was wrong about needing to specify different levels for different sizes. The alert header gets the same style regardless of the heading level. Duh. 🤦